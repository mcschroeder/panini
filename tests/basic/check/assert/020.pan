assert : { b:𝔹 | b = true } → 𝟙

f20 : unit
f20 = assert false

f21 : {v:unit|true}
f21 = assert false

f22 : {v:unit|v=unit}
f22 = assert false
