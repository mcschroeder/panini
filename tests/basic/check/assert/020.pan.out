tests/basic/check/assert/020.pan:4:1-4: error: unable to verify type of f20
tests/basic/check/assert/020.pan:7:1-4: error: unable to verify type of f21
tests/basic/check/assert/020.pan:10:1-4: error: unable to verify type of f22